src/fragments/lib/logging/ios/enable_disable/enable_disable.mdx

```diff
@@ -0,0 +1,11 @@
+Enable logging via the Amplify Logging category.
```

Comment:
In this section, you will learn how to enable and disable logging when using the Amplify Logger.

Reply:
Added additional details and updated title.

Reply:
I just feel like the rest of libraries keep the code snippets and the imports all together to make it easier to copy and paste things. We should be consistent. We'll be making those changes across for all of our guides soon.